packages/zendesk-adapter/src/change_validators/guide_theme_readonly.ts

```diff
@@ -30,8 +30,8 @@ export const guideThemeReadonlyValidator: ChangeValidator = async changes => {
 
   return themeInstances.map(instance => ({
     elemID: instance.elemID,
-    severity: 'Error',
-    message: 'Cannot deploy this element because Guide Themes is currently under development.',
-    detailedMessage: 'Guide themes is in read only mode and not deployable at the moment.',
+    severity: 'Warning',
+    message: 'Guide Themes is currently under development.',
+    detailedMessage: 'The Guide Themes feature is currently under development. Please note that references to other elements will not function correctly upon deployment',
```

Comment:
nit: maybe
```suggestion
    detailedMessage: 'Guide Themes deploy support is currently under development. Please note that Theme code referencing other elements might not transfer correctly between environments',
```
(can also discuss IRL, either way not a blocker for merging)